src/harness/certs/README.md

```diff
@@ -1,15 +1,55 @@
 This directory contains the certificates used by the test harness to connect
 with the SAS under test.
 
-The certificates which are checked into GitHub are merely placeholders. SAS
-administrators should replace them with valid certificates before testing.
+NOTE: Before running the tests you must provide the certificates described in
+this file:
+- The shell script `generate_fake_certs.sh` can be used to generate fake version
+  to be used with the `fake_sas.py` implementation.
+- To run against a real SAS, please generate your own certificates.
 
-* client.[cert|key] - used in all tests not concerned with security-related
-  features and tests utilizing a domain proxy
+The various certificates follow the PKI structure defined on the "CBRS COMSEC TS
+WINNF-15-S-0065" document. Naming and base configuration are issued from
+https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/cert
 
-* ca.cert - used by the test harness to authenticate the SAS under test
+```
+                               root_ca
+                               /     \
+                              /       \
+                         sas_ca       cbsd_ca
+                         /    |         |
+                        /     |         |
+             admin_client  server   client+device_xxx
+```
 
-* admin_client.[cert|key] - used to authenticate the test harness when
-  connecting to the SAS testing API
+Refer to the `generate_fake_certs.py` script and `../../cert/openssl.cnf` file
+to get command line example on how to generate required certificates and keys.
 
+Required certificates are:
```

Comment:
It probably makes sense to add device_[a|c].[cert|key] to this list and to generate_fake_certs.py.